documentation-samples/multinic/azuredeploy.json

```diff
@@ -0,0 +1,287 @@
+      "defaultValue": "NSG-RemoteAccess",
+      "metadata": {
+        "description": "Name for the NSG used to allow remote access."
+      }
+    },
+    "stdStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstoragestd",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageName": {
+      "type": "string",
+      "defaultValue": "wtestvnetstorageprm",
+      "metadata": {
+        "description": "Name for simple storage account."
+      }
+    },
+    "prmStorageType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS",
+      "allowedValues": [
+        "Premium_LRS",
+        "Premium_GRS",
+        "Premium_ZRS"
+      ]
+    },
+    "osType": {
+      "type": "int",
+      "defaultValue": 0,
+      "allowedValues": [
+        0,
+        1
+      ],
+      "metadata": {
+        "description": "Type of OS to use for VMs: 0 - Windows; 1 - Ubuntu."
+      }
+    },
+    "dbCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of database VMs to be deployed to the backend end subnet."
+      }
+    },
+    "parentRG": {
+      "type": "string",
+      "defaultValue": "IaaSStory",
+      "metadata": {
+        "description": "Name of resource group containing vnet and front end resources."
+      }
+    }
+  },
+  "variables": {
+    "dbVMSettings": [
+      {
+        "vmSize": "Standard_DS3",
+        "adminUsername": "adminuser",
+        "adminPassword": "adminP@ssw0rd",
+        "publisher": "MicrosoftSQLServer",
+        "offer": "SQL2014SP1-WS2012R2",
+        "sku": "Standard",
+        "version": "latest",
+        "vmName": "DB",
+        "osdisk": "osdiskdb",
+        "datadisk": "datadiskdb",
+        "nicName": "NICDB",
+        "ipAddress": "192.168.2.",
+        "extensionDeployment": "",
+        "avsetName": "ASDB",
+        "remotePort": 3389,
+        "dbPort": 1433
+      },
+      {
+        "vmSize": "Standard_DS3",
+        "adminUsername": "adminuser",
+        "adminPassword": "adminP@ssw0rd",
+        "publisher": "Canonical",
+        "offer": "UbuntuServer",
+        "sku": "14.04.2-LTS",
+        "version": "latest",
+        "vmName": "DB",
+        "osdisk": "osdiskdb",
+        "datadisk": "datadiskdb",
+        "nicName": "NICDB",
+        "ipAddress": "192.168.2.",
+        "extensionDeployment": "",
+        "avsetName": "ASDB",
+        "remotePort": 22,
+        "dbPort": 1433
+      }
+    ],
+    "location": "[resourceGroup().location]",
+    "vmStorageAccountContainerName": "vhds",
+    "vnetID": "[resourceId(parameters('parentRG'),'Microsoft.Network/virtualNetworks',parameters('vnetName'))]",
+    "backEndSubnetRef": "[concat(variables('vnetID'),'/subnets/',parameters('backEndSubnetName'))]",
+    "dbVMSetting": "[variables('dbVMSettings')[parameters('osType')]]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
```

Comment:
Not entirely sure if this will work with VS but take a look here https://github.com/Azure/azure-resource-manager-schemas/tree/master/schemas